src/main/java/listeners/MainEventListener.java

```diff
@@ -253,7 +257,43 @@ private void showQueue(Member member, TextChannel channel, Server server, String
         }
     }
 
-    private void clear(Member member, TextChannel channel, Server server, String[] args) {
+    private void kick(Member member, TextChannel channel, Server server, String[] args, Member[] mentions) {
+        Member mentee = mentions[0];  // also takes up args[0]
+        String topicName = args[1];
+        String reason = Stream.of(args).skip(2).collect(Collectors.joining(" "));
+
+        // do not run if topic does not exist
+        Optional<Topic> optionalTopic = checkTopicExists(member, channel, server, topicName);
+        if (optionalTopic.isEmpty()) return;
+
+        // do not run if caller does not have mentor role for this topic or admin privileges
+        Topic topic = optionalTopic.get();
+        if (!isMentor(member, topic) && !isAdmin(member)) {
+            channel.sendMessage(member.getAsMention() + " You do not have permission to run this command.").queue();
+            return;
+        }
+
+        // do not run if mentee is not in the specified queue
+        if (!topic.isInQueue(mentee)) {
+            channel.sendMessage(String.format(
+                "%s User \"%s\" is not in the queue for topic \"%s\".",
```

Comment:
In future might be good to factor these to -- oh wait, @sjmiller7 are these the messages you're currently factoring out to a static class?

Reply:
Yep, they are